### PR TITLE
New example for gallery and some docs refinements

### PR DESCRIPTION
**Pull request by kolibril13**
New example with ValueTracker and GraphScene, and some docs enhancements in the Text and Tex section
See here:
https://manimce--1415.org.readthedocs.build/en/1415/reference/manim.mobject.svg.tex_mobject.html#module-manim.mobject.svg.tex_mobject

Depends on #1472

### CODE REVIEW

File: docs/source/examples.rst
Comment (kolibril13) [inline docs/source/examples.rst:678]: oh, stupid copy paste mistake

**Comment by Darylgolden** [comment]
We're planning to deprecate `GraphScene` with #1161; please update your example to use Axes. Perhaps it would be better to wait until that PR gets merged.

**Review by behackl — CHANGES_REQUESTED** [review]
`GraphScene` will be deprecated soon, the example should rather use `Axes` instead. It might make sense to make this PR dependent on #1161.

**Comment by kolibril13** [comment]
As all of the other scenes in the example gallery also use GraphScene , https://docs.manim.community/en/latest/examples.html#plotting-with-manim
do we want to change all of them at once? (so, merging this pr in its current state, and changing it to Axes in a future pr.)

**Comment by behackl** [comment]
No, we should change as soon as possible. There is no benefit in waiting to change until later (except that the work only has to be done later).

I sort of want the updated examples to be included in the new release; I'll investigate how easy it is to change them.

**Comment by kolibril13** [comment]
Sounds good!
I also think it could be done before the release, but as all the other examples have to be updated as well, why not merging this pr now and then merging the next pr before the release?
Otherwise, when you find a way to update the other examples quickly, feel free to commit directly to this branch :)

File: docs/source/examples.rst
Comment (kolibril13) [inline docs/source/examples.rst:636]: I removed SineCurveUnitCircle, as it is a bit too long as an example on the main example gallery and not programmed in the most efficient way.

Comment (behackl) [reply docs/source/examples.rst:636]: If you say that it wasn't programmed in the most efficient way, the example should be improved -- but I disagree with completely removing it. I think it makes sense for our example gallery to include a few longer / more complex examples as well.

Comment (kolibril13) [reply docs/source/examples.rst:636]: e.g. the part :
```py
def add_x_labels(self):
            x_labels = [
                MathTex("\pi"), MathTex("2 \pi"),
                MathTex("3 \pi"), MathTex("4 \pi"),
            ]

            for i in range(len(x_labels)):
                x_labels[i].next_to(np.array([-1 + 2*i, 0, 0]), DOWN)
                self.add(x_labels[i])
```
I don't like too much, and I currently don't have the time to rewrite it by myself.
Also, I think it does not add too much value to the manim feature overview.
Instead, we could add some other classes, e.g. the Graph class or an example to show how to use rate functions.

Comment (behackl) [reply docs/source/examples.rst:636]: > it does not add too much value to the manim feature overview

I disagree. It shows how you can combine several concepts to construct a full-fledged animation.

I don't have time to rewrite it either, and there is still a lot of work to do for the release. I'm happy to keep discussing this, but I feel that this change is too controversial to be merged quickly.

Comment (kolibril13) [reply docs/source/examples.rst:636]: sounds good.
And I also I agree that we should show a full-fledged animation, but maybe not exactly this one.
But this can be discussed after the release.

File: manim/mobject/svg/tex_mobject.py
Comment (naveen521kk) [inline manim/mobject/svg/tex_mobject.py:4]: I don't think the module name is necessary. Same for the other change also.
```suggestion
   Just as you can use :class:`~.Text` to add text to your videos, you can use :class:`~.Tex` and :class:`~.MathTex` to insert LaTeX.
```

Comment (kolibril13) [reply manim/mobject/svg/tex_mobject.py:4]: I think it is good to have, as the text_mobject module is right now really difficult to find in the docs.
And when one wants to use tex, one might probably also see some example of how to use text.
Also, it makes it more clear, that tex is not part of of the text_mobject

**Review by naveen521kk — APPROVED** [review]

**Comment by hydrobeam** [comment]
I've updated the example to use `Axes` with some minor changes/fixes, although I realized that I accidentally submitted the changes as a commit instead of a PR. If you'd like to revert any of the changes, feel free to do so.

**Comment by kolibril13** [comment]
> I accidentally submitted the changes as a commit instead of a PR

totally fine, actually I prefer it this way -> less steps for me :)

So this pr depends then on #1472?

**Comment by hydrobeam** [comment]
> So this pr depends then on #1472?

Yup, I was working on restructuring some examples that use `GraphScene` from that branch which reworked some parameters then I happened to notice this PR. Since the method is going to eventually be readjusted with that PR I decided to leave it as is.

For the sake of allowing the adjusted example to be seen, here's the rendered image in comparison to the previous one:

Old:
image
New:
image

File: docs/source/examples.rst
Comment (hydrobeam) [inline docs/source/examples.rst:322]: This string feels a little overwhelming to include (both visually and in terms of code) especially since it's intended for the example gallery. Perhaps it'd be better to remove or adjust it?

Comment (kolibril13) [reply docs/source/examples.rst:322]: Yeah, good point!
My initial idea was that this example shows on the one hand a mathematical function, and on the other hand the transfer of it to in manim.
But you are right, it can get removed, as this example gallery should also be for people who do not study math :)

Comment (kolibril13) [inline docs/source/examples.rst:322]:
```suggestion

```

**Review by behackl — APPROVED** [review]

**Pull request merged**
